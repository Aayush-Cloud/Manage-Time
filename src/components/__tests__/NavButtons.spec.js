// @flow
import React from 'react';
import renderer from 'react-test-renderer';
import {
  AddButton,
  CloseButton,
} from '../NavButtons';

it('renders AddButton correctly', () => {
  const tree = renderer.create(
    <AddButton onPress={() => {}} />,
  ).toJSON();
  expect(tree).toMatchSnapshot();
});

it('renders CloseButton correctly', () => {
  const tree = renderer.create(
    <CloseButton onPress={() => {}} />,
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
