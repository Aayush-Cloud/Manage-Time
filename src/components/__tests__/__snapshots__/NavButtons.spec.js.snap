// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders AddButton correctly 1`] = `
<View
  accessibilityComponentType={undefined}
  accessibilityLabel={undefined}
  accessibilityTraits={undefined}
  accessible={true}
  hitSlop={
    Object {
      "bottom": 12,
      "left": 12,
      "right": 12,
      "top": 12,
    }
  }
  isTVSelectable={true}
  onLayout={undefined}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "opacity": 1,
    }
  }
  testID={undefined}
  tvParallaxProperties={undefined}
>
  <Text
    accessible={true}
    allowFontScaling={false}
    ellipsizeMode="tail"
    style={
      Array [
        Object {
          "color": "white",
          "fontSize": 24,
        },
        undefined,
        Object {
          "fontFamily": "Material Icons",
          "fontStyle": "normal",
          "fontWeight": "normal",
        },
      ]
    }
  >
    
  </Text>
</View>
`;

exports[`renders CloseButton correctly 1`] = `
<View
  accessibilityComponentType={undefined}
  accessibilityLabel={undefined}
  accessibilityTraits={undefined}
  accessible={true}
  hitSlop={
    Object {
      "bottom": 12,
      "left": 12,
      "right": 12,
      "top": 12,
    }
  }
  isTVSelectable={true}
  onLayout={undefined}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "opacity": 1,
    }
  }
  testID={undefined}
  tvParallaxProperties={undefined}
>
  <Text
    accessible={true}
    allowFontScaling={false}
    ellipsizeMode="tail"
    style={
      Array [
        Object {
          "color": "white",
          "fontSize": 24,
        },
        undefined,
        Object {
          "fontFamily": "Material Icons",
          "fontStyle": "normal",
          "fontWeight": "normal",
        },
      ]
    }
  >
    
  </Text>
</View>
`;
